tests/istanbul/test_state_transition.py

```diff
@@ -0,0 +1,135 @@
+from functools import partial
+from typing import Dict
+
+import pytest
+
+from ethereum.exceptions import InvalidBlock
+from tests.helpers.load_state_tests import (
+    Load,
+    fetch_state_test_files,
+    idfn,
+    run_blockchain_st_test,
+)
+
+fetch_istanbul_tests = partial(fetch_state_test_files, network="Istanbul")
+
+FIXTURES_LOADER = Load("Istanbul", "istanbul")
+
+run_istanbul_blockchain_st_tests = partial(
+    run_blockchain_st_test, load=FIXTURES_LOADER
+)
+
+# Run legacy general state tests
+test_dir = "tests/fixtures/BlockchainTests/GeneralStateTests/"
+
+# Every test below takes more than  60s to run and
+# hence they've been marked as slow
+SLOW_TESTS = (
+    "stTimeConsuming/CALLBlake2f_MaxRounds.json",
+    "stTimeConsuming/static_Call50000_sha256.json",
+    "vmPerformance/loopExp.json",
+    "vmPerformance/loopMul.json",
+    "QuadraticComplexitySolidity_CallDataCopy_d0g1v0_Istanbul",
+    "CALLBlake2f_d9g0v0_Istanbul",
+    "CALLCODEBlake2f_d9g0v0",
+)
+
+# These are tests that are considered to be incorrect,
+# Please provide an explanation when adding entries
+INCORRECT_UPSTREAM_STATE_TESTS = (
+    # The test considers a scenario that cannot be reached by following the
+    # rules of consensus. For more details, read:
+    # https://github.com/ethereum/py-evm/pull/1224#issuecomment-418775512
+    "stRevertTest/RevertInCreateInInit.json",
+    # The test considers a scenario that cannot be reached by following the
+    # rules of consensus.
+    "stCreate2/RevertInCreateInInitCreate2.json",
+    # The test considers a scenario that cannot be reached by following the
+    # rules of consensus.
+    "stSStoreTest/InitCollision.json",
+    "chainId_d0g0v0_Istanbul",  # TODO: remove after EIP-1344
+    "chainIdGasCost_d0g0v0_Istanbul",  # TODO: remove after EIP-1344
+    "badOpcodes_d21g0v0_Istanbul",  # TODO: remove after EIP-1344
```

Comment:
Just leaving myself a note to check these.

Reply:
These have been removed from the list and are being executed normally as EIP-1344 has been implemented.